internal/app/configuration/proxyconfig.go

```diff
@@ -13,6 +13,9 @@ func NewFromEnv() (pactproxy.Config, error) {
 	ctx := context.Background()
 
 	var config pactproxy.Config
+
+	config.RejectUnrecognizedInteractions = true
```

Comment:
It feels like it might be more safe to flip the logic here. i.e. have the flag be `AllowUnrecognizedInteractions`.
That way the effect of not setting the field at all is the same as the existing functionality, and this won't be a breaking change if (for whatever reason) a caller is manually constructing a `pactproxy.Config` instead of using `NewFromEnv`.

Reply:
I think that's better, I went ahead and made the change.